tests/customers/tests-customers-db.php

```diff
@@ -217,42 +217,4 @@ public function test_legacy_remove_payment_should_return_false() {
 
 		$this->assertFalse( in_array( $payment_id, $payment_ids ) );
 	}
-	
-	public function test_legacy_increment_stats_purchase_value_should_return_10() {
-		EDD()->customers->increment_stats( self::$customers[0], 10 );
-
-		/** @var $customer \EDD_Customer */
-		$customer = edd_get_customer( self::$customers[0] );
-		
-		$this->assertSame( 10.0, $customer->purchase_value );
-	}
-
-	public function test_legacy_increment_stats_purchase_count_should_return_1() {
-		EDD()->customers->increment_stats( self::$customers[0], 10 );
-
-		/** @var $customer \EDD_Customer */
-		$customer = edd_get_customer( self::$customers[0] );
-
-		$this->assertSame( 1, $customer->purchase_count );
-	}
-
-	public function test_legacy_decrement_stats_purchase_value_should_return_90() {
-		EDD()->customers->increment_stats( self::$customers[0], 100 );
-		EDD()->customers->decrement_stats( self::$customers[0], 10 );
-
-		/** @var $customer \EDD_Customer */
-		$customer = edd_get_customer( self::$customers[0] );
-
-		$this->assertSame( 90.0, $customer->purchase_value );
-	}
-
-	public function test_legacy_decrement_stats_purchase_count_should_return_0() {
-		EDD()->customers->increment_stats( self::$customers[0], 10 );
-		EDD()->customers->decrement_stats( self::$customers[0], 10 );
-
-		/** @var $customer \EDD_Customer */
-		$customer = edd_get_customer( self::$customers[0] );
-
-		$this->assertSame( 0, $customer->purchase_count );
-	}
-}
```

Comment:
It's a little bit unusual to delete unit tests, even for formally deprecated functions & methods. It would be nice to know that these deprecated methods still work the way they used to in perpetuity.

Reply:
The tests are deleted per [this comment](https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7947#issuecomment-641486431).

Reply:
I expect new tests may need to be written but I do not know how.

Reply:
Unit tests are fuuuuuuuuuuuuunnnn! 🥳

It's completely OK to not know how, and it's also completely OK to not want to write them once you do know how. We can talk privately about them more and come up with a plan. 🌻